diff --git a/node_modules/dmg-builder/out/dmgUtil.js b/node_modules/dmg-builder/out/dmgUtil.js
index f6f3e95..c70f2d6 100644
--- a/node_modules/dmg-builder/out/dmgUtil.js
+++ b/node_modules/dmg-builder/out/dmgUtil.js
@@ -162,8 +162,17 @@ async function customizeDmg({ appPath, artifactPath, volumeName, specification,
         settings.background = specification.background == null ? null : await transformBackgroundFileIfNeed(specification.background, packager.info.tempDirManager);
     }
     if (!(0, builder_util_1.isEmptyOrSpaces)(settings.background)) {
-        const size = await getImageSizeUsingSips(settings.background);
-        settings.window = { position: { x: 400, y: Math.round((1440 - size.height) / 2) }, size, ...settings.window };
+        const imageSize = await getImageSizeUsingSips(settings.background);
+        const window = specification.window;
+        const size = {
+            width: (window === null || window === void 0 ? void 0 : window.width) || imageSize.width,
+            height: (window === null || window === void 0 ? void 0 : window.height) || imageSize.height,
+        };
+        const position = {
+            x: (window === null || window === void 0 ? void 0 : window.x) || 400,
+            y: (window === null || window === void 0 ? void 0 : window.y) || Math.round((1440 - size.height) / 2),
+        };
+        settings.window = { position, size, ...settings.window };
     }
     const settingsFile = await packager.getTempFile(".json");
     await (0, fs_extra_1.writeFile)(settingsFile, JSON.stringify(settings, null, 2));
